File: src/test/java/com/Johnny/student/StudentTest.java
package com.Johnny.student;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {

    @Test
    void highestScoreTest() {
        Student student = new Student("Johnny", 60, 80 ) ;
        Assertions.assertEquals(80,student.highest());
    }

    @Test
    void averageTest() {
        Student student = new Student("Johnny", 60, 80 ) ;
        Assertions.assertEquals((60+80)/2, student.getAverage());
    }
}

File: src/test/kotlin/com/Johnny/StudentTest.kt
package com.Johnny.kotlin

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test


class StudentTest {
    @Test
    fun highestTest(){
        val student = Student( "Johnny", 60, 80 )
        Assertions.assertEquals(80, student.hightest())
    }

    @Test
    fun averageTest() {
        val student = Student( "Johnny", 60, 80 )
        Assertions.assertEquals((60+80)/2, student.getAverage())

    }

    @Test
    fun graddingTest() {

        val student = Student( "Johnny", 60, 80 )
        Assertions.assertEquals('C', student.gradding())
    }
}